corehq/apps/app_manager/views/settings.py

```diff
@@ -66,3 +71,30 @@ def edit_commcare_profile(request, domain, app_id):
     response_json = {"status": "ok", "changed": changed}
     app.save(response_json)
     return json_response(response_json)
+
+
+class PromptSettingsUpdateView(FormView, ApplicationViewMixin):
+    form_class = PromptUpdateSettingsForm
+    urlname = 'update_prompt_settings'
+
+    @property
+    def success_url(self):
+        return reverse('release_manager', args=[self.domain, self.app_id])
+
+    @method_decorator(no_conflict_require_POST)
+    @method_decorator(require_can_edit_apps)
```

Comment:
if you want to avoid having to override the method you can do this:
```
@method_decorator(require_can_edit_apps, name='dispatch')
class PromptSettingsUpdateView(...):

```